workitem/link/link_repository.go

```diff
@@ -111,6 +113,10 @@ func (r *GormWorkItemLinkRepository) Create(ctx context.Context, sourceID, targe
 		}
 		return nil, errors.NewInternalError(db.Error.Error())
 	}
+	// save a revision of the created work item link
+	if err := r.revisionRepo.Create(ctx, creatorID, RevisionTypeCreate, *link); err != nil {
+		return nil, errs.WithStack(err)
```

Comment:
What's the point with wrapping it here since the `link.GormWorkItemLinkRevisionRepository` already returns this error:
````
if err := tx.Create(&revision).Error; err != nil {
	return errors.NewInternalError(fmt.Sprintf("Failed to create new work item link revision: %s", err.Error()))
}
````
What kind of wrapping should I add ?